src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -18,15 +18,18 @@ import NavPageLink from './NavPageLink';
 import DrawerHeader from '../../components/DrawerHeader';
 import { ChevronLeft, ChevronRight } from '@mui/icons-material';
 import NERDrawer from '../../components/NERDrawer';
-import { LayoutProps } from '../LayoutProps';
+import { GridMenuIcon } from '@mui/x-data-grid';
+import { useState } from 'react';
 
-export interface SideBarProps extends LayoutProps {
-  handleDrawerClose: () => void;
+interface SidebarProps {
+  open?: boolean;
```

Comment:
I literally just added that so that the tests would work. The sidebar tests depend on having the sidebar be open, but I couldn't figure out if there was a way to have like vi or something open the sidebar after it was created, so I made a way to create the sidebar in an open state.

Reply:
Other solutions include 1) having the test open the sidebar after its created, 2) deleting the sidebar tests

Reply:
mmm I see, let's change the name to defaultOpen then and have it function as such. I wanted this component to function as an uncontrolled component, an open prop which effectively controls the open state would make it more controlled. Usually it's better to stick to one or the other (not that we do a great job of that in the codebase). Some reading if you're interested: https://www.geeksforgeeks.org/controlled-vs-uncontrolled-components-in-reactjs/